inst/CITATION

```diff
@@ -5,6 +5,6 @@ citEntry(
   title    = "manyigos",
   author   = "Hollway, James",
   year     = "2022",
-  note = "For more information on the qVerse: https://github.com/globalgov/manydata"
+  note = "For more information on the manyverse: https://github.com/globalgov/manydata",
   textVersion = paste("Hollway, James", "manyigos", "2022", sep = ".")
```

Comment:
Commenting on the logo here:

A good first version. The isomorphism doesn't look quite right though. I would also suggest that we find a different colour for it. The only grayscale logos should be for manydata and manypkgs, so that they stand out. What would be a nice contrast colour to what we already have? Orange?